src/query_parser/tokenizer.h

```diff
@@ -2,8 +2,9 @@
 #define __QUERY_TOKENIZER_H__
 
 #include <stdlib.h>
+#include <stdbool.h>
 #include "../tokenize.h"
-#include "vector_index.h"
+#include "VecSim/vec_sim_common.h"
```

Comment:
why the change here?

Reply:
There was cyclic include:
tokenizer.h -> vector_index.h -> query_ctx.h -> query_param.h -> tokenizer.h
Actually now that I look, we shouldn't include query_param.h from query_ctx.h. 
But anyway, I think that it is safer that tokenizer.h will include vec_sim_common.h (to avoid cyclic includes in the future), no?

Reply:
I included `vector_index.h` for `VecSimRawParam` but since we moved this struct to the library we can change this. On the other hand maybe we want to include files from the library through `vector_index.h` anyway